src/containers/forgot-password/ForgotPassword.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { Button, Form, Input, Typography } from 'antd';
+import authClient from '../../auth/authClient';
+import { ForgotPasswordRequest } from '../../auth/ducks/types';
+
+const { Title } = Typography;
+
+const ForgotPassword: React.FC = () => {
+  const onFinish = (values: ForgotPasswordRequest) => {
+    authClient
+      .forgotPassword(values)
+      .then(() => {
+        alert('Successfully sent forgot password request!');
```

Comment:
We should probably use an AntD alert for this if possible

Reply:
We didn't include this in our most recent commit bc we figured that people might not necessarily want to have an alert. An alternative would just be displaying some text on screen or something